### ISSUE TITLE
Need USB + HW serial configuration in menu.

### ISSUE THREAD

**Issue by terrillmoore** [mcci-catena/Arduino_Core_STM32#70]
Looks like in fixing #29 we broke an important use case: USB connection combined with local serial port (as Serial1). It probably is possible to work around this in the sketch, but we ought to support this as an option.

**Comment by AlexSneedMiller** [comment]
Until this is fixed, does this mean that you cannot have a serial port? Or is it just that if you have the USB cable plugged in then the board will not be able to have a serial port on different pins?

**Comment by terrillmoore** [comment]
It's reltively easy to hack this in your boards.txt -- it's just the combination of #defines. In fact, the new 4630 board has serial (for the PM2.5 sensor) and USB. if you look (for example) the 4612, you'll see:

```
mcci_catena_4612.menu.xserial.usb.build.xSerial={build.usb_flags} -DNO_HWSERIAL
```

If you change that to:

```
mcci_catena_4612.menu.xserial.usb.build.xSerial={build.usb_flags} -DFIRST_THIRD_HWSERIAL
```

Then you'll get both usbserial and UARTS 1 and 3 enabled.  There may be some confusion about which port `Serial` refers to (as opposed to `UsbSerial` and `Serial1`), but otherwise things should work. In an older version, they did work.

**Comment by terrillmoore** [comment]
Fixed by #98.
